lib/tool-bar-button-view.js

```diff
@@ -15,11 +15,12 @@ export default class ToolBarButtonView {
     this.group = group;
 
     if (options.tooltip) {
-      this.element.title = options.tooltip;
```

Comment:
Note: if there's a title attribute on the DOM node, Atom will use it and not respect either title or keyBindingCommand passed as attribute.

It's doing weird things with the title here: https://github.com/atom/atom/blob/master/src/tooltip.js#L349-L354